CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+0.6.2 / February 10 2017
```

Comment:
best to say "unreleased" - requiring enzyme 2.3 is technically a breaking change, so this would at a minimum be 0.7.0, but it's not for individual PRs to decide version numbers.